tests/test_trusted-execution.yml

```
@@ -0,0 +1,132 @@
+---
+- name: Basic sanity test for fapolicyd
+  hosts: localhost
+  vars:
+    binaries_directory: "/var/tmp/executable_binaries"
+    source_directory: "{{ binaries_directory }}/source"
+    source_file: "{{ source_directory }}/main.c"
+    exe1: "{{ binaries_directory }}/exe1"
+    exe2: "{{ binaries_directory }}/exe2"
+    user: "testuser"
+
+  tasks:
+    - name: Create {{ binaries_directory }} directory
+      file:
+        path: "{{ binaries_directory }}"
+        state: directory
+        mode: '0755'
+
+    - name: Create {{ source_directory }} directory
+      file:
+        path: "{{ source_directory }}"
+        state: directory
+        mode: '0755'
+
+    - name: Create C source code (binary1)
+      copy:
+        content: |
+          int main(void) { 
+              return 0; 
+          }
+        dest: "{{ source_file }}"
+
+    - name: Install build essentials (for RHEL-based systems)
+      block:
+        - name: Install GCC
+          yum:
+            name: gcc
+            state: present
+
+        - name: Install glibc-devel (for standard library headers)
+          yum:
+            name: glibc-devel
+            state: present
+    
+    - name: Compile C programs
+      command: "gcc -o {{ exe1 }} {{ source_file }}"
```

```suggestion
      command: gcc -o {{ exe1 }} {{ source_file }}
```

Quotes are not necessary.